src/parameters/all_parameters.cpp

```diff
@@ -90,6 +90,14 @@ void AllParameters::declare_parameters (dealii::ParameterHandler &prm)
                       dealii::Patterns::Bool(),
                       "Use other boundary conditions by default. Otherwise use periodic (for 1d burgers only");
 
+    prm.declare_entry("renumber_dof_handler_Cuthill_Mckee", "true",
+                      dealii::Patterns::Bool(),
+                      "Renumber dof handler with Cuthill-Mckee by default. Don't renumber dof_handler if false.");
+
+    prm.declare_entry("use_curvilinear_grid", "false",
+                      dealii::Patterns::Bool(),
+                      "Use straight grid by default. Curvilinear is true.");
```

Comment:
```suggestion
                      "Use straight grid by default. Curvilinear is true. Only used in taylor_green_scaling test.");
```
I think this should eventually be moved out of `all_parameters` as it is a test-specific parameter. If we ever move the curvilinear TGV tests into flow solver, we should set it there. For now, clarifying where it is used is good enough for me.

Reply:
Ideally, there would be a mesh specific parameters where you could choose which type of grid and all of the grid parameters. For now will use your suggested commit.